soccer/src/soccer/planning/planner_node.hpp

```diff
@@ -75,8 +75,10 @@ class SharedStateInfo {
             });
     }
 
+    // TODO: do these need mutex locks?
     [[nodiscard]] PlayState play_state() const {
-        auto lock = std::lock_guard(mutex_);
+        auto lOCK = std::lock_guard(mutex_);
+
         return last_play_state_;
```

Comment:
Yes it should have be locked: https://preshing.com/20130618/atomic-vs-non-atomic-operations/.